src/mono/sample/wasm/blazor-frame/wwwroot/frame.js

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+"use strict";
+
+import { dotnet, exit } from './_framework/dotnet.js'
```

Comment:
Why do we need separate import?

Reply:
not anymore, that remained from previous version